airline-web/app/models/Chat.scala

```diff
@@ -12,7 +14,12 @@ final case class ClientSentMessage(text: String)
 // Chat actor
 class Chat extends Actor {
   // initial message-handling behavior
-  var log = Queue[String]()
+  
+  // General Chat Log (50)
+  var g_log = Queue[String]()
```

Comment:
should be `val`?

Reply:
don't think so as it's constantly modified, might as well keep it mutable.

Reply:
the Queue is mutable but not the reference i suppose?

Reply:
changed it